tests/run_tests.py

```diff
@@ -220,18 +221,20 @@ def run_tests(tests: list[Path], workers: int) -> int:
 
 
 def build_jit_dependencies() -> None:
-    runtime_src_path = PARENT_DIR.parent / "std" / HOST_TARGET / "runtime.S"
-    assert runtime_src_path.is_file()
-
-    # Don't compile the runtime again if it's up-to-date.
-    if (
-        not RUNTIME_OBJ_PATH.is_file()
-        or runtime_src_path.stat().st_mtime > RUNTIME_OBJ_PATH.stat().st_mtime
-    ):
-        # Need to use `cc` because `as` doesn't run the C preprocessor.
-        subprocess.run(
-            ["cc", "-c", runtime_src_path, "-o", RUNTIME_OBJ_PATH], check=True
-        )
+    with resources.as_file(
+        resources.files("glang.lib.std") / HOST_TARGET / "runtime.S"
```

Comment:
Huh, cool. I've never seen this... This looks like something from native app development not python.

Reply:
Yeah... I really wanted this to work so that we can split our source tree, but it doesn't really do what I wanted... It's more effort than it's worth